README.md

```diff
@@ -49,24 +49,29 @@ These scripts have been tested with Python 3.9.1(latest).
 
 To use these scripts, please follow the following steps:
 
-1. **Generate a personal access token**	
+1. **Add SSH keys to your GitHub account**
+
+   - Follow the steps on this website: https://www.inmotionhosting.com/support/server/ssh/how-to-add-ssh-keys-to-your-github-account
```

Comment:
github also has an official guide: [https://docs.github.com/en/github/authenticating-to-github/adding-a-new-ssh-key-to-your-github-account](https://docs.github.com/en/github/authenticating-to-github/adding-a-new-ssh-key-to-your-github-account) .. might be good to add it

Reply:
That's better, thank you